### ISSUE TITLE
Missing docs for cilium uninstall

### ISSUE THREAD

**Issue by joebowbeer** [cilium/cilium#17098]
## Bug report

In cilium-cli v0.8.5 there is an `uninstall` command but no `cleanup` command:

https://github.com/cilium/cilium-cli/blob/master/internal/cli/cmd/install.go#L98

Whereas in the cilium docs (v1.10), there is a `cleanup` command but no `uninstall` command:

https://github.com/cilium/cilium/blob/master/Documentation/cmdref/cilium_cleanup.md

**Comment by gandro** [comment]
Thanks for the report. I'm marking this as a question instead of a bug, because there are two binaries called `cilium`:

One is the Cilium CLI hosted in https://github.com/cilium/cilium-cli/, which is used to install and validate Cilium on new clusters. It contains cluster-wide operations for e.g. connectivity validation or uninstalling Cilium from a cluster.
This binary is typically installed on workstations. It has a `uninstall` command, but no `cleanup` command.

The other binary is also called `cilium` but it is only used inside the Docker container to configure the _local_ `cilium-agent` instance. It has a `cilium cleanup` command which removes all Cilium-related state from the local node - but if you want to uninstall Cilium from a cluster, that's not sufficient, you will have to use `cilium uninstall` for that.

To summarize:

- The `cilium` binary available alongside the `cilium-agent` on each node has its source here (https://github.com/cilium/cilium/tree/master/cilium) and is documented here (https://docs.cilium.io/en/v1.10/cmdref/cli_index/)
- The `cilium` binary called the Cilium CLI is has it's source here (https://github.com/cilium/cilium-cli/) and currently has no hosted documentation.

**Comment by gandro** [comment]
We have since renamed the `cilium` binary inside of the cilium-agent pod to `cilium-dbg`, to avoid further confusion with "Cilium CLI".

All documentation should have been updated. `cleanup` is a command for `cilium-dbg` which is only available inside of the cilium-agent pod.
